packages/SwingSet/test/bootstrap-relay.js

```diff
@@ -46,18 +54,46 @@ export const buildRootObject = () => {
     },
 
     /**
+     * Intended to be called via `messageVat` (encoding done by our caller).
      * Turns an object into a remotable by ensuring that each property is a function
      *
      * @param {string} label
      * @param {Record<string, any>} methodReturnValues
+     * @param {object} [options]
+     * @param {boolean} [options.logCalls] make call logs available via `getLogForRemotable`
      */
-    makeSimpleRemotable: (label, methodReturnValues) =>
-      // braces to unharden so it can be hardened
-      encodePassable(
-        Far(label, {
-          ...objectMap(methodReturnValues, v => () => decodePassable(v)),
+    makeRemotable: (label, methodReturnValues, { logCalls = false } = {}) => {
```

Comment:
Is there any reason not to always log calls? This is ephemeral testing code, so memory consumption shouldn't be a concern.